### PR TITLE
[Fixes #116 & #131] Changes done

### LINKED ISSUE #116: Pytorch Learning Rate should be float
## Bug Report
Not a very important bug as of now

### Current Behavior
Learning rate is int, increases by 1 

### Expected behavior/code
Should be float supporting 10e-9 in the worst case

### LINKED ISSUE #131: Homescreen bug
## Bug Report

Logout button visible in homepage without being logged in.

### Current Behavior
homepage

### Expected behavior
Check login status to display logout button  in side navbar

### PR DESCRIPTION
**Pull request by rajraj889**
## Pull Request

fixes #131 
fixes #116 

### Checklist:
- [x] Followed the guidelines mentioned in our [Contributing document](/CONTRIBUTING.md)
- [x] Ensured that there aren't other open [Pull Requests](https://www.github.com/Auto-DL/Generator/pulls) for the same update/change
- [x] This PR is linked to an [Issue](https://www.github.com/Auto-DL/Generator/issues) and the issue number has been mentioned in the title and body of the PR
- [x] The code is clean and docstrings have been added or updated as required
- [x] Code is linted/formatted locally prior to submission (using `black`)

### CODE REVIEW

**Review by ADI10HERO — APPROVED** [review]

**Comment by ADI10HERO** [comment]
image

This happened when I clicked the side nav on the login page...
And on a ctrl-shift-R the side-nav was gone, so was there supposed to be a side nav initially in the first place?
